### DISCUSSION TITLE
Download timed out at 66% - Not sure how to proceed

### DISCUSSION THREAD

**Discussion by Sparks01** [icloud-photos-downloader/icloud_photos_downloader#672] [Q&A]
The download seemed to time out or break at around 66%. I got about  200 GB of a 245B library. I'm not sure what to do to grab the rest. I can't tell what it left out. iCloud says I have 48,000 images but the DL only grabbed  39K. Do I need to start over? 

Thx. 

EDIT : I just read below about a similar issue. Was unaware the icloudopd will not duplicate files. So I can just run it over and over until complete? Thanks.

**Comment by boredazfcuk** [comment]
> So I can just run it over and over until complete?

Yes. It will 'fill in the gaps' and not re-download what it has already downloaded.
